Content.Server/GameObjects/Components/Mobs/Body/Limb.cs

```diff
@@ -0,0 +1,322 @@
+﻿using System;
+using Robust.Shared.Maths;
+using System.Collections.Generic;
+using Robust.Shared.Serialization;
+using Content.Shared.GameObjects;
+using Robust.Shared.Log;
+using Robust.Shared.Interfaces.GameObjects;
+using YamlDotNet.RepresentationModel;
+using Robust.Shared.Prototypes;
+using Content.Server.GameObjects.Components.Mobs.Body.Organs;
+using Robust.Shared.IoC;
+using System.Linq;
+using Robust.Shared.Utility;
+using Robust.Shared.Interfaces.Reflection;
+using Robust.Shared.Reflection;
+
+namespace Content.Server.GameObjects.Components.Mobs.Body
+{
+    /// <summary>
+    ///     Limb is not just like <see cref="Organ"/>, it has BONES, and holds organs (and child limbs), 
+    ///     it receive damage first, then through resistances and such it transfers the damage to organs,
+    ///     also the limb is visible, and it can be targeted
+    /// </summary>
+    [Prototype("limb")]
+    public class Limb : IPrototype, IIndexedPrototype
```

Comment:
1. Once again, mutable prototype.
2. A lot of the code between limbs and organs seems to be shareable, just like SS13's external/internal organ distinction. You should probably give them a common base like SS13 does to reduce duplicate code.